.azure-pipelines-templates/trace_validation.yml

```diff
@@ -9,5 +9,6 @@ steps:
   - script: |
       set -ex
       cd tla/
-      parallel 'JSON={} ./tlc.sh consensus/Traceccfraft.tla' ::: $(ls ../build/*.ndjson | grep -v _deprecated)
+      for f in ../build/*.ndjson; do jq -c 'del(.cmd)' $f > $f.tmp && mv $f.tmp $f; done
```

Comment:
I'll replace null with "" to avoid the slowdown and the need for this line. I need to make a couple of tweaks for retirement anyway.

Reply:
Done now.
```suggestion
```

Reply:
OK, I will go ahead with the proposed fix in https://github.com/tlaplus/tlaplus/issues/868 to fail fast, i.e., reject `null`.